packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/openshift/openshiftSecretFieldValidator.ts

```diff
@@ -41,7 +41,7 @@ export const openshiftSecretFieldValidator = (id: string, value: string): Valida
 const validateToken = (value: string): ValidationMsg => {
   const valid = validateK8sToken(value);
 
-  if (value === '') {
+  if (value === undefined || value === '') {
     return {
       type: 'default',
       msg: 'A service account token, required for authenticating the the connection to the API server.',
```

Comment:
```suggestion
      msg: 'A service account token, optional, used for authenticating the the connection to a remote API server.',
```

see comment https://github.com/kubev2v/forklift-console-plugin/pull/1117/files#r1570429644